### DISCUSSION TITLE
Wrong path paths in .css (result of export Next.js)

### DISCUSSION THREAD

**Discussion by viktor-mozdolevskyi** [vercel/next.js#30948] [Help]
I faced the next issue. As the result of the next.js export, I receive wrong paths in CSS files, for example:
```
background: url(/images/oval_21.webp)
```
but should be
```
background: url(../../../images/oval_21.webp)
```
The same thing happens with fonts. Also, I need to note images in public folder 'public/images' and I am using 'next-optimized-images' for handling images in export, and here is my config setup:
```
module.exports = withPlugins([
  [optimizedImages, {
    optimizeImages: false,
  }],
  {
    assetPrefix: isDeployMode ? '.' : '',
    reactStrictMode: true,
    experimental: { esmExternals: true },
    env: {
      CQ_USE_ANALYTICS: JSON.stringify(config.useAnalytics),
      CQ_AMPLITUDE_API_KEY: JSON.stringify(config.amplitudeApiKey),
    },
    images: {
      domains: ["localhost"],
    },
  }
]);
```
Tried without 'next-optimized-images' and got the same result.

Can anyone help me with this or faced this issue before?

**Comment by vidforstner** [comment]
I have the same issue with fonts. Anyone found an answer yet? Couldn't find anything useful myself.
